core/model/modx/processors/workspace/packages/getlist.class.php

```diff
@@ -1,190 +1,190 @@
-<?php

```

Comment:
Sorry for the noise on that one...
That's just CR/LF change automatically performed by my IDE while i was playing around with this processor...